docs/platform-external/installing.md

```diff
@@ -0,0 +1,471 @@
+      # Wait for NetworkManager to report it's online
+      After=NetworkManager-wait-online.service
+      # Run before kubelet
+      Before=kubelet.service
+      [Service]
+      # Mark afterburn environment file optional, due to it is possible that afterburn service was not executed
+      EnvironmentFile=-/run/metadata/afterburn
+      ExecStart=/usr/local/bin/kubelet-providerid
+      Type=oneshot
+      [Install]
+      WantedBy=network-online.target
+```
+
+The `{{ machine_role }}` must be `master` and `worker`. Both manifests must be saved to `config-master.bu` and `config-worker.bu`, then run the following commands:
+
+```bash
+$ butane config-master.bu \
+  -o openshift/99_openshift-machineconfig_00-master-kubelet-providerid.yaml
+
+$ butane config-worker.bu \
+  -o openshift/99_openshift-machineconfig_00-worker-kubelet-providerid.yaml
+```
+
+#### Create additional manifests
+
+If the CCM or any other component you want to add in install time requires dependencies, you can also add it to the respective directories.
+
+For example, if the CCM requires secrets or custom configuration, you can create it:
+
+> replace the `{{ secret_value | b64encode }}` to a valid base64 string
+
+```bash
+cat <<EOF > manifests/cloud-controller-manager-01-secret.yaml
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: cloud-controller-manager-config
+data:
+  cloud-provider.yaml: {{ secret_value | b64encode }}
+EOF
+```
+
+### Create ignition files
+
+Once the manifests are placed, you can create the cluster ignition configurations:
+
+~~~bash
+./openshift-install create ignition-configs
+~~~
+
+The files with the extension `.ign` must be generated.
+
+```text
+.
+├── auth
+│   ├── kubeadmin-password
+│   └── kubeconfig
+├── bootstrap.ign
+├── master.ign
+├── metadata.json
+└── worker.ign
+```
+
+## Create compute nodes
+
+The step to provision compute nodes must be cloud-specific. The required steps to boot RHCOS is different for each compute role:
+
+- bootstrap node: must use the ignition file `bootstrap.ign` to initialize
+- control-plane nodes: must use the ignition file `master.ign`
+- compute nodes: must use the ignition file `worker.ign`
+
+The steps below describe each one.
+
+Requirements:
+
+- You created the ignition configs
+- You uploaded the RHCOS image to the cloud image service
+- You reviewed the ["Minimum resource requirements for cluster installation"](https://docs.openshift.com/container-platform/4.13/installing/installing_platform_agnostic/installing-platform-agnostic.html#installation-minimum-resource-requirements_installing-platform-agnostic)
+
+### Bootstrap
+
+The `bootstrap.ign` must be used to create the bootstrap node. Most of the cloud providers have limitations on the user-data size, so you must store the `bootstrap.ign` externally, and retrieve it in the boot process. If the cloud provides the blob service allowing to create of a signed HTTPS URL, it can be used in the ignition file as follow:
```

Comment:
Maybe?:
```suggestion
The `bootstrap.ign` must be used to create the bootstrap node. Most of the cloud providers have size limits in the user data, so you must store the `bootstrap.ign` externally, then retrieve it in the cloud-init process. If the cloud provider has blob service allowing to create of a signed HTTPS URL, it can be used to store and serve the ignition file. The cloud-init could be used as a as follow:
```

Reply:
the rewrite looks nice, but the last line needs adjustment: `The cloud-init could be used as follows:"